docs/details/signal.dox

```diff
@@ -19,8 +19,8 @@ batch mode convolutions take place.
 - **Identical Batches** - A set of filters applied onto to a set of inputs in one-to-one correspondence.
 - **Non overlapping Batches** - All batched filters are applied to all batched signals. The batch dimension of Signal and Filter **should not** be the same.
 
-Non overlapping batch mode is not supported in spatial mode i.e. if the user passes \ref
-AF_CONV_SPATIAL explicitly to the functions.
+\note All Non-overlapping(Interleaved) convolutions default to frequency domain
```

Comment:
I  think those words are proper adjectives. Hence, initial letter can be capitalized. I am changing it any way since I am not sure about it.

The `Dims` is completely removed. In any case, I have intentionally kept them apart earlier as they are related to different sections.